jekyll/_cci2/release/set-up-circleci-releases.adoc

```diff
@@ -0,0 +1,73 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Set up CircleCI releases
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to set up CircleCI releases, covering creating an environment integration, installing the release agent and creating components.
+:icons: font
+:experimental:
+
+Visualise and control deployments to your Kubernetes cluster or SageMaker service with CircleCI releases. The steps outlined on this page guide you to set up CircleCI releases using the CircleCI web app onboarding process. Alternatively, once you have set up an environment integration, you can follow the guides linked below to install the release agent and configure components manually.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will set up an integration between CircleCI and your environment (Kubernetes cluster or SageMaker). Once configured, release status can be reported to the CircleCI web app, and the following commands can be executed in your cluster or service by the release agent via controls in the CircleCI UI:
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+Before setting up your environment integration with CircleCI, run through the following prerequisites:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A Kubernetes cluster or Amazon SageMaker service
```

Comment:
I see we talk here about SageMaker but the rest of this guideline is tailored to Kubernetes, should we remove SageMaker from here or refer to an external guideline to Sagemaker specifically